tests/lowering/tensor_manipulation/test_reshape.py

```diff
@@ -12,10 +12,9 @@ def forward(self, x, new_shape):
         return torch.reshape(x, new_shape)
 
 
-@pytest.mark.xfail(reason="lowering issue (#67)")
 @pytest.mark.parametrize(
     "input_shape, new_shape",
-    [((3 * 5, 7), (7 * 3, 5))],
+    (((3 * 5, 7), (7 * 3, 5)), ((1, 512, 1, 1), (1, 512))),
```

Comment:
This case passes on its own.  Maybe we need to check the next op or the synergy between `reshape` and the next op?